root/etc/s6-overlay/s6-rc.d/init-config-ml/run

```diff
@@ -3,34 +3,33 @@
 mkdir -p /config/machine-learning
 
 # apply early ml support
-cp /defaults/main.py /app/immich/machine-learning/src/main.py
+cp /defaults/main.py /app/immich/machine-learning/main.py
 if [ ! -d /config/cuda-venv ]; then
     python3 -m venv /config/cuda-venv/
 fi
 
 # install/update pip packages to /config venv
 source /config/cuda-venv/bin/activate
 
-pip install -U --no-cache-dir --index-url https://download.pytorch.org/whl/cu117 \
-    torch
-
 pip install -U --no-cache-dir \
+    aiocache \
     fastapi \
     insightface \
     nltk \
     numpy \
     onnxruntime-gpu \
-    packaging \
     Pillow \
     scikit-learn \
     scipy \
     sentencepiece \
     sentence-transformers \
+    toch \
     tqdm \
     transformers \
     uvicorn[standard]
 
 # permissions
 chown -R abc:abc \
+    /photos \
     /config \
     /app
```

Comment:
@hydazz do we need to change ownership for `/app` ?

Reply:
not unless any services write to it, also `toch`?

Reply:
better with a `r`🤦‍♂️